wtpython/displays/textual_display.py

```diff
@@ -73,10 +73,13 @@ def check_overflow(contents: list[Text], console: Console, size: Size) -> bool:
             Returns whether the sidebar is overflowing or not
         """
         page = Text(end="")
-        for content in contents:
+        for i, content in enumerate(contents):
             page.append_text(content)
-            page.append_text(Text("\n\n"))
-        page.append_text(Text("<- Prev Page 0/0 Next ->"))
+            if i != len(contents) - 1:
+                page.append_text(Text("\n\n"))
+        page.append_text(Text("3cksjusdkcjsady"))
```

Comment:
Should we hardcode this? Or maybe generate something random programmitcally? Just feels weird to me. Also this is a Stackoverflow specific method, should it be in Stackoverflow parts of the code rather than here?